Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -114,12 +114,12 @@ For the best experience with the Slider Widget, we recommend adding multiple ***
 
 ![movePage.jpg](images/The-Tabs-Widget.png)
 
-This allows you to create multiple widgets within one widget this means you could have for example: a ***Text Widget***, an ***Image Widget*** and a ***Forms Widget*** all collected in one.
+The Tabs Widget allows you to create tabs like the ones you find in your browser. Each tab can contain any widgets you desire.
+You can even add multiple widgets within one ***Tabs Widget*** which means you could have the ***Text Widget*** and the ***Image Widget*** on tab 1 and ***Forms Widget*** is tab2, because you can have Multiple tabs in one ***Tabs widget***. 
```

Comment:
Removed the last bit, as you already mention that in the beginning of the sentence.

```suggestion
You can even add multiple widgets within one ***Tabs Widget***. This means you could have the ***Text Widget*** and the ***Image Widget*** on one tab and a ***Forms Widget*** on another tab.
```